lib/utils.ts

```diff
@@ -26,5 +26,105 @@ export const getNameString = (authors: string[] | string) => {
     return authors.join();
   }
 
+  if (size > 3) {
+    return authors.slice(0, 2).join(', ') + ' and ' + (size - 2) + ' others';
+  }
+
   return authors.slice(0, size - 1).join(', ') + ' and ' + authors[size - 1];
```

Comment:
I think it would be better to wrap this one around `if size === 2`.

Reply:
Yes, makes a lot more sense. Done!

Reply:
I think you forgot this :thinking: